### PR TITLE
Adds guard against null being passed where non null is expected.

### PR DESCRIPTION
**Pull request by valeriyvan**
Solves issue #2

### CODE REVIEW

File: RubikSolver-iOS/RubikSolver-iOS/ViewControllers/CubeSolverViewController.mm [hunk 1]
```diff
@@ -433,20 +433,22 @@ - (IBAction)didPressPreviousButton:(UIButton *)sender {
...
-            [self.scene3D.rootNode addChildNode:child];
-        }];
-        
-        [rotationNode removeFromParentNode];
-        
-        self.nextButton.enabled = YES;
-        self.previousButton.enabled = YES;
-    }];
+
+	if (rotationSCNAction) {
```
Comment (rhcpfan) [inline RubikSolver-iOS/RubikSolver-iOS/ViewControllers/CubeSolverViewController.mm]: Can it be `if (!rotationSCNAction) return;` in order to reduce code nesting?

Comment (valeriyvan) [reply RubikSolver-iOS/RubikSolver-iOS/ViewControllers/CubeSolverViewController.mm]: Sure, early return looks better here.

Comment (rhcpfan) [reply RubikSolver-iOS/RubikSolver-iOS/ViewControllers/CubeSolverViewController.mm]: Ok, update the pull request then, so I can close it 👍

**Review by rhcpfan — APPROVED** [review]

**Comment by rhcpfan** [comment]
Thanks a lot! 👍

**Pull request merged**